apps/agentic-chat/src/tools/searchTokens.ts

```diff
@@ -62,7 +62,10 @@ export const searchTokens = async ({
 
       const assetId = toAssetId({
         chainId,
-        assetNamespace: 'erc20',
+        assetNamespace:
```

Comment:
would be nice to not bring this along (should really remove this concept in web as well).

Reply:
You mean, consider all ERC20s and not do the bep20 dance? Actually that sounds super sane. 
And for BSC AssetId, we could redeclare it as `erc20` chainNamespace instead, WDYT? 

Alternatively, we could do a two birds one stone here and tackle across the board in web, then publish and bump, and wouldn't even need to redeclare a new erc20 `bnbAssetId`, since it would already be like that in web/caip package.

Reply:
Ideally both places I think. I added to be exact to spec initially, but it provides literally zero value. Let's prep this as needed without waiting for the caip update and then we can follow up in web, publish and follow up here.

Reply:
Disregard me am derp, only thing we needed was to remove the bep20 stuff in `toAssetId()` and voila d30c45c98f93e7c05526a26937c61a29783236d5

<img width="731" alt="image" src="https://github.com/user-attachments/assets/aebd1d16-2207-4a5a-b2a5-e968d16c1344" />